modules/cassandra/options.go

```diff
@@ -0,0 +1,127 @@
+package cassandra
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+// TLSConfig represents the TLS configuration for Cassandra
+type TLSConfig struct {
```

Comment:
question: are the paths needed? If not we can just return `*tls.Config`. I suspect the answer is dependent on eliminating the need for `keytool`